zk/model/config.py

```diff
@@ -0,0 +1,37 @@
+""" Configuration key store, elements such as event_city or contact_email """
+import sqlalchemy as sa
+from sqlalchemy import orm
+from sqlalchemy.dialects.postgresql import JSON # Required - unsure why
+from meta import Base, Session, metadata
+
+import logging
+log = logging.getLogger(__name__)
+
+class Config(Base):
+    __tablename__ = 'config'
+    category = sa.Column(sa.types.String, primary_key=True)
+    key      = sa.Column(sa.types.String, primary_key=True)
+    value    = sa.Column(sa.dialects.postgresql.JSON)
+    # NOTE: Non-postgresql requires the use of a decorator
+
+    @classmethod
+    def get(cls, category, key=None):
```

Comment:
There are recipes for what they call dogpile caching.
It is commonly tied up with the session life, which would be less useful, but I don't think it has to be.
Another approach talks about using memcached but I don't think we need to go that far.

I expect the work will be more around having a mechanism to flush or update the cache when values are changed and doing some benchmarking to demonstrate that there is actually a performance gain.

My preference to delay it is that this merge will stand alone without the caching and it is rather large and ugly as it is (while also being as small as realistically possible). Adding caching would be a nice clean independent merge, as would a basic CRUD gui page.